L2 Understading electric vehicless/Identifying core and secondary parts of a Battery Electric Vehicle (BEV).md

```diff
@@ -95,24 +97,24 @@ When the vehicle is charged from a home electrical grid (e.g. 220 V),
 
 1. The **rectifier** converts the alternating current (AC) into direct current (DC), which is fed into the high voltage battery. 
 
-2. The **DC-DC converter** is responsible with the lowering of the high voltage (e.g. 400 V) to the low voltage network (12 V).
+2. The **DC-DC converter** is responsible for the lowering of the high voltage (e.g. 400 V) to the low voltage network (12 V).
 
-3. The **inverter** controls the electric machine speed and torque by converting the direct current from the battery into the alternating 3-phase current for the electric machine. 
+3. The **inverter** controls the electric machine's speed and torque by converting the direct current from the battery into the alternating 3-phase current for the electric machine. 
 
 4. When the vehicle is in the energy recuperation phase (braking) the inverter is doing the opposite conversion, from 3-phase AC to DC.
 
 5. The **input filter** reduces the electric noise or interference generated during regenerative braking, when energy is fed back into the batteries via the motors.
 
-##S Understanding motors
+### Understanding motors
```

Comment:
```suggestion
## Understanding motors
```

Reply:
Noted, I'll be checking headings properly.